packages/authentication/src/__tests__/acceptance/basic-auth.acceptance.ts

```diff
@@ -1,208 +0,0 @@
-// Copyright IBM Corp. 2019. All Rights Reserved.
```

Comment:
Are we removing these tests because they are already covered/moved to some other place (Perhaps @emonddr's PR had them)?

Reply:
@b-admike These tests are written using a passport based strategy + passport adapter. While the old adapter will be deprecated and replaced by a new one to be created in a new module, so **acceptance test for the passport strategy** should be rewritten in that coming new module.

The new auth sysmte that Dom created is built with the extension point, so that the new acceptance tests also reflect that change:
see https://github.com/strongloop/loopback-next/blob/master/packages/authentication/src/__tests__/acceptance/jwt-auth-extension.acceptance.ts
and https://github.com/strongloop/loopback-next/blob/master/packages/authentication/src/__tests__/acceptance/basic-auth-extension.acceptance.ts 

> Are we removing these tests because they are already covered/moved to some other place
So to answer your question, yes or no, the acceptance tests for the new auth system is covered. The new adapter and its corresponding acceptance tests haven't created. The file you are looking at is removed because of the old adapter doesn't work with the new system anymore.

Reply:
Makes sense, thanks for the in-depth explanation @jannyHou :-).